L1Trigger/L1THGCal/src/backend/HGCalShowerShape.cc

```diff
@@ -5,6 +5,9 @@
 
 #include <unordered_map>
 
+HGCalShowerShape::HGCalShowerShape(const edm::ParameterSet& conf)
+    : threshold_(conf.existsAs<double>("shape_threshold") ? conf.getParameter<double>("shape_threshold") : 0.) {}
```

Comment:
we typically prefer to avoid `existsAs`. Can this parameter be added to any configs that are missing it?